.github/workflows/ci.yml

```diff
@@ -43,6 +43,16 @@ jobs:
       - run: ./scripts/coverage.sh
         shell: bash
 
+  build_image:
+    name: Build Docker Image
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - run: echo ${{ secrets.docker_pass }} | docker login --username ${{ secrets.docker_username }} --password-stdin
```

Comment:
This uses the same setup as AvalancheGo, ran into issues with the docker login GitHub action where I wasn't able to get it working: https://github.com/docker/login-action

Reply:
should we keep the echo? not sure where these logs end up